app/src/main/java/org/mozilla/focus/home/HomeTilePlaceholderGenerator.kt

```diff
@@ -12,12 +12,14 @@ import android.net.Uri
 import android.support.v4.content.ContextCompat
 import android.util.TypedValue
 import org.mozilla.focus.R
+import org.mozilla.focus.ext.withRoundedCorners
 import org.mozilla.focus.utils.UrlUtils
 
 class HomeTilePlaceholderGenerator {
 
     companion object {
-        private val TEXT_SIZE_DP = 52f
+        private val TEXT_SIZE_DP = 22f
+        private val DEFAULT_TILE_CORNER_RADIUS = 25f
```

Comment:
While I'm not entirely opposed to leaving this in the file, we should be adjusting this to DP (like how textSize does it) otherwise the corner rounding amount could change per device.